File: public/phase-2-20160620022203/PYTHON/notebook-with-numbers.py
#!/usr/bin/python
# Name : Notebook with Numbers 
# input:
# n : number of sheets
# k : 
# a : 
elements = raw_input().strip().split(' ')
n = int(elements[0])
k = int(elements[1])
a_elements = raw_input().strip().split(' ')
a = map(int, a_elements)


# write your code here
# store your results in `maximum_sum`,`r`

# output

# Dummy Data
maximum_sum = 27
r = [3,4,5,8]

print(maximum_sum)
print(" ".join(str(e) for e in r))

File: public/phase-2-20160620022203/C/notebook-with-numbers.c
// Name : Notebook with Numbers 
#include <stdio.h>
int main() {
	// input:
	// n : number of sheets
	// k : 
	// a : 
	int n, k, a[300], maximum_sum=27, r[300]={3,4,5,8}, index;
	scanf("%d %d", &n, &k);
	for(index = 0; index< n; index++) {
		scanf("%d", &a[index]);
	}


	// write your code here
	// store your results in `maximum_sum`,`r`

	// output
	printf("%d", maximum_sum);
	for(index = 0; index< (k-1); index++) {
		printf("%d ",r[index]);
	}
	return 0;
}